komodo/release_transpiler.py

```diff
@@ -204,20 +204,31 @@ def main():
     matrix_parser.add_argument(
         "--release-folder",
         required=True,
+        type=dir_path,
         help="Folder with existing release file (default: None)",
     )
     matrix_parser.add_argument(
         "--override-mapping",
         required=True,
+        type=check_if_valid_file,
         help="File containing explicit matrix packages (default: None)",
     )
+
+    def checkValidPythonVersion(python_versions: str) -> list:
+        output_list = []
+        for item in python_versions.split(","):
+            if re.match(r"^[2,3](\.\d+)?$", item) is None:
```

Comment:
If we just want to require a parseable version string, we might use `packaging.version.parse()` instead of maintaining our own regexp (doomed to fail for Python 4 :) )